django/db/backends/mysql/client.py

```diff
@@ -11,7 +11,7 @@ def settings_to_cmd_args(cls, settings_dict, parameters):
         args = [cls.executable_name]
         db = settings_dict['OPTIONS'].get('db', settings_dict['NAME'])
         user = settings_dict['OPTIONS'].get('user', settings_dict['USER'])
-        passwd = settings_dict['OPTIONS'].get('passwd', settings_dict['PASSWORD'])
```

Comment:
Now when we use the same names we could simplified this (in a separate commit) with:
```python
conn_params = {**settings_dict, settings_dict['OPTIONS']}
db = conn_params.get('db', '')
user = conn_params.get('user', '')
....
```

Reply:
Yes, that would be better, I will update. Also, could you please tell me where to add these release notes? I couldn't find any info in the patch submitting guide. Sorry.

Reply:
https://docs.djangoproject.com/en/3.0/internals/contributing/writing-code/submitting-patches/#patch-review-checklist

Reply:
Thanks. Just to be sure, do I need to add release note only in `docs/releases/3.1.txt` or in the older files (3.0 etc) as well?

Reply:
Only in 3.1, we don't backport cleanups.

Reply:
PR updated, kindly review.